column-samples/generic-yearly-gantt-chart/generic-yearly-gantt-chart-with-today.json

```diff
@@ -149,15 +149,15 @@
         "height": "8px"
       },
       "attributes": {
-        "class":"ms-fontColor-themePrimary"
+        "class": "ms-fontColor-themePrimary"
       },
       "children": [
         {
           "elmType": "div",
           "style": {
             "position": "relative",
             "width": "= '1' + '%'",
-            "left": "= floor(((Number(@now)-Number(Date('' + getYear([$Start]) + '/1/1')))/(1000*60*60*24))/365 * 100) + '%'"
+            "left": "= floor(((Number(@now)-Number(Date('' + getYear(@now) + '/1/1')))/(1000*60*60*24))/365 * 100) + '%'"
```

Comment:
Modifying [$Start] to @now, for example, would cause a red line to appear representing today, even if the schedule is for next year.
How about reverting back to the previous state so that only this year's schedule shows the red line?
![image](https://user-images.githubusercontent.com/51254139/197367128-ef67e28b-c9d1-452b-9501-4bc273880e43.png)